### PR TITLE
Feature/evidence aatf approval

### PR DESCRIPTION
**Pull request by JohnPadgettCivica**
For https://dev.azure.com/civica-digital/EA%20-%20WEEE/_workitems/edit/51583

### CODE REVIEW

**Review by meglenapnovoselska — APPROVED** [review]

File: src/EA.Weee.Web/Areas/Aatf/Attributes/EvidenceNoteEndDateAttribute.cs [hunk 1]
```diff
@@ -30,23 +25,24 @@ protected override ValidationResult IsValid(object value, ValidationContext vali
             var thisDate = ((DateTime)value).Date;
             var otherDate = (DateTime?)validationContext.ObjectType.GetProperty(CompareDatePropertyName)?.GetValue(validationContext.ObjectInstance, null);
 
-            if (CheckComplianceYear)
+            if (!WindowHelper.IsDateInComplianceYear(thisDate.Year, currentDate))
```
[the remark this answers is not in the document]
Comment (JohnPadgettCivica) [reply src/EA.Weee.Web/Areas/Aatf/Attributes/EvidenceNoteEndDateAttribute.cs:28]: Which line?

[the remark this answers is not in the document]
Comment (meglenapnovoselska) [reply src/EA.Weee.Web/Areas/Aatf/Attributes/EvidenceNoteEndDateAttribute.cs:28]: I meant if thisDate is a start date then is the validation message correct return new ValidationResult("The end date must be within the current compliance year");?

[the remark this answers is not in the document]
Comment (JohnPadgettCivica) [reply src/EA.Weee.Web/Areas/Aatf/Attributes/EvidenceNoteEndDateAttribute.cs:28]: ah no. In this attribute thisDate is the endDate and  the otherDate is the compare to date which is the start date,...

[the remark this answers is not in the document]
Comment (meglenapnovoselska) [reply src/EA.Weee.Web/Areas/Aatf/Attributes/EvidenceNoteEndDateAttribute.cs:28]: okay, leave it then

**Pull request merged**